variables.tf

```
@@ -76,6 +76,12 @@ variable "enable_internet_gateway" {
   default     = true
 }
 
+variable "enable_egress_only_internet_gateway" {
```

We have started using this new naming convention

```suggestion
variable "egress_only_internet_gateway_enabled" {
```